Doc/tutorial/modules.rst

```diff
@@ -225,17 +225,18 @@ version of spam.py would be cached as ``__pycache__/spam.cpython-33.pyc``.  This
 naming convention allows compiled modules from different releases and different
 versions of Python to coexist.
 
-Python checks the modification date of the source against the compiled version
-to see if it's out of date and needs to be recompiled.  This is a completely
-automatic process.  Also, the compiled modules are platform-independent, so the
-same library can be shared among systems with different architectures.
+Python checks the modification date of the source (``.py``) against the compiled
+(``.pyc``) version to see if it's out of date and needs to be recompiled.  This
```

Comment:
Lines 223-224 just above have an example of `spam.py` and `spam.cpython-33.pyc`; I think it would be better to work the term _source_ there.